js/bitmex.js

```diff
@@ -43,6 +43,7 @@ module.exports = class bitmex extends Exchange {
                 'fetchLedger': true,
                 'fetchLeverage': false,
                 'fetchLeverageTiers': false,
+                'fetchMarginMode': false,
```

Comment:
It seems that they actually do have a way of fetching a margin mode and probably the position mode as well.

@sc0Vu please, check the following and redo this PR:

<img width="1367" alt="Screenshot 2022-06-16 at 13 06 11" src="https://user-images.githubusercontent.com/1294454/174047435-7a72c283-185a-4fac-b47e-82b58f82dc7b.png">
<img width="1361" alt="Screenshot 2022-06-16 at 13 06 41" src="https://user-images.githubusercontent.com/1294454/174047482-0134ad67-f6a8-4b26-86e9-463bbff8a8d4.png">